bundles/org.openhab.binding.homematic/src/main/resources/OH-INF/i18n/homematic.properties

```diff
@@ -54,3 +60,8 @@ thing-type.config.homematic.bridge.xmlCallbackPort.description = Callback port o
 
 channel-type.homematic.DUTY_CYCLE_RATIO.label = Duty Cycle
 channel-type.homematic.DUTY_CYCLE_RATIO.description = Current duty cycle usage
+
+# thing types config
+
+thing-type.config.homematic.bridge.username.label = Username for accessing the gateway
+thing-type.config.homematic.bridge.username.description = Username for accessing the gateway. Is required if the "useAuthentication" is true
```

Comment:
Did you run the i18n plugin or was it manually added? This keys allready exist around line 52.

Reply:
I run i18n plugin - however i have a look on this